frontend/src/lib/utils.ts

```diff
@@ -5,6 +5,8 @@ const questionServiceBackendUrl =
 	import.meta.env.VITE_QUESTION_SERVICE_BACKEND_URL || "http://localhost:5002";
 const collabServiceBackendUrl =
 	import.meta.env.VITE_COLLAB_SERVICE_BACKEND_URL || "http://localhost:5004";
+const userServiceBackendUrl =
+  import.meta.env.VITE_USER_SERVICE_BACKEND_URL || "http://localhost:5001";
```

Comment:
@ekweirui I have updated the files through the latest commit, do help me check and verify I did it right . You might have to comment out `collabServiceBackendUrl` in utils.ts since this branch doesn't use it and npm won't let you build with unused variables.